packages/create-sitecore-jss/src/templates/nextjs-sxa/src/lib/middleware/plugins/redirects.ts

```diff
@@ -24,6 +24,8 @@ class RedirectsPlugin implements MiddlewarePlugin {
       disabled: () => process.env.NODE_ENV === 'development',
       // Site resolver implementation
       siteResolver,
+      // This list of sites need for searching context site language when we use token like $siteLang it's not locale of site
+      sites: JSON.parse(config.sites),
```

Comment:
No need to add this config. `SiteResolver` class has a readonly `sites` prop you can use (i.e. use `this.config.siteResolver.sites`).

Here's an example where we use it:
https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/nextjs-multisite/src/lib/sitemap-fetcher/plugins/graphql-sitemap-service.ts#L19

Reply:
Thank you